Realm/Tests/PropertyTests.m

```diff
@@ -139,4 +144,18 @@ - (void)testSwiftPropertyNameValidation {
     XCTAssertNoThrow(RLMValidateSwiftPropertyName(@"__alloc"));
 }
 
+- (void)testTypeToString {
```

Comment:
This test is pretty dubious since it's not really going to catch anything other than a typo in RLMTypeToString, but it is sorta testing something.